.github/workflows/test.yml

```diff
@@ -77,30 +83,20 @@ jobs:
     strategy:
       matrix:
         elixir:
-        - "1.7"
-        - "1.8"
         - "1.9"
         - "1.10"
         - "1.11"
         otp:
-        - "19.3"
-        - "20.0"
         - "21.0"
-        - "22.0"
+        - "22.0.2"
         - "23.0"
-        exclude:
-        - elixir: 1.8
-          otp: 19.3
-        - elixir: 1.9
-          otp: 19.3
-        - elixir: 1.10
-          otp: 19.3
-        - elixir: 1.10
-          otp: 20.0
-        - elixir: 1.11
-          otp: 19.3
-        - elixir: 1.11
-          otp: 20.0
+        include:
+        - elixir: "1.7"
+          otp: "19.3"
+        - elixir: "1.8"
+          otp: "20.0"
+        - elixir: "1.9"
+          otp: "20.0"
```

Comment:
🙋 By changing this from a blacklist to a whitelist, many important pairs have been missed. I don't think this change is necessarily beneficial but am interested to hear more about why you preferred it.

If you do want to make this change from `exclude` to `include`, then the missing version pairs are needed. For example, Elixir 1.10 and 1.11 have disappeared completely.